### PR TITLE
docs(forms): add text about min() and max() as functions

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #18830

## What is the new behavior?
Adds a comment that `min` and `max` are functions, not directives.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/forms/src/validators.ts [hunk 1]
```diff
@@ -75,6 +75,7 @@ const EMAIL_REGEXP =
 export class Validators {
   /**
    * Validator that requires controls to have a value greater than a number.
+   *`min()` exists only as a function, not as a directive.
```
Comment (kara) [inline packages/forms/src/validators.ts]: Maybe clarify a bit to add suggested usage? Same for the other one.

```
... exists only as a validator function, not a directive. e.g.:

control = new FormControl('', Validators.min(3));

```

Comment (kapunahelewong) [reply packages/forms/src/validators.ts]: Thank you @kara! I've added this for each.

**Comment by IgorMinar** [comment]
@kapunahelewong can you please change the scope in the commit message from "aio" to "forms"? "aio" is just for angular.io application changes and this is a document content change so we should label it by the component it affects ("forms" in this case)

**Review by kara — APPROVED** [review]
LGTM (pending Igor's requested commit change)

**Comment by kapunahelewong** [comment]
Commit message changed and rebased. Thank you, @IgorMinar!

**Comment by IgorMinar** [comment]
@kapunahelewong did you forget to force push to the upstream branch? I don't see the changed commit message.

**Comment by IgorMinar** [comment]
I see. you changed the pull request subject and not the commit message.

to change the commit message you have to do the following:

```
git checkout wong-min-max
git commit --amend
# edit the commit message and save
git push origin wong-min-max --force-with-lease
```

**Comment by kapunahelewong** [comment]
@IgorMinar Oh! Sorry! I'm with you now. Thank you! :)

**Pull request closed without merging**